src/AppBundle/Controller/Admin/VoteSettingsController.php

```diff
@@ -0,0 +1,246 @@
+     */
+    public function editAction($id)
+    {
+        $em = $this->getDoctrine()->getManager();
+
+        $entity = $em->getRepository('AppBundle:VoteSettings')->find($id);
+
+        if (!$entity) {
+            throw $this->createNotFoundException('Unable to find VoteSettings entity.');
+        }
+
+        $editForm = $this->createEditForm($entity);
+        $deleteForm = $this->createDeleteForm($id);
+
+        return array(
+            'entity'      => $entity,
+            'edit_form'   => $editForm->createView(),
+            'delete_form' => $deleteForm->createView(),
+        );
+    }
+
+    /**
+    * Creates a form to edit a VoteSettings entity.
+    *
+    * @param VoteSettings $entity The entity
+    *
+    * @return \Symfony\Component\Form\Form The form
+    */
+    private function createEditForm(VoteSettings $entity)
+    {
+        $form = $this->createForm(new VoteSettingsType(), $entity, array(
+            'action' => $this->generateUrl('vote_settings_update', array('id' => $entity->getId())),
+            'method' => 'PUT',
+        ));
+
+        $form->add('submit', 'submit', array('label' => 'Update'));
+
+        return $form;
+    }
+    /**
+     * Edits an existing VoteSettings entity.
+     *
+     * @Route("/{id}", name="vote_settings_update")
+     * @Method("PUT")
+     * @Template("AppBundle:VoteSettings:edit.html.twig")
+     */
+    public function updateAction(Request $request, $id)
+    {
+        $em = $this->getDoctrine()->getManager();
+
+        $entity = $em->getRepository('AppBundle:VoteSettings')->find($id);
+
+        if (!$entity) {
+            throw $this->createNotFoundException('Unable to find VoteSettings entity.');
+        }
+
+        $deleteForm = $this->createDeleteForm($id);
+        $editForm = $this->createEditForm($entity);
+        $editForm->handleRequest($request);
+
+        if ($editForm->isValid()) {
+            $em->flush();
+
+            return $this->redirect($this->generateUrl('vote_settings_edit', array('id' => $id)));
+        }
+
+        return array(
+            'entity'      => $entity,
+            'edit_form'   => $editForm->createView(),
+            'delete_form' => $deleteForm->createView(),
+        );
+    }
+    /**
+     * Deletes a VoteSettings entity.
+     *
+     * @Route("/{id}", name="vote_settings_delete")
+     * @Method("DELETE")
+     */
+    public function deleteAction(Request $request, $id)
+    {
+        $form = $this->createDeleteForm($id);
+        $form->handleRequest($request);
+
+        if ($form->isValid()) {
+            $em = $this->getDoctrine()->getManager();
+            $entity = $em->getRepository('AppBundle:VoteSettings')->find($id);
+
+            if (!$entity) {
+                throw $this->createNotFoundException('Unable to find VoteSettings entity.');
+            }
+
+            $em->remove($entity);
+            $em->flush();
+        }
+
+        return $this->redirect($this->generateUrl('vote_settings'));
```

Comment:
In each not GET Method Action you return form View, but not here. What is happen? Your code doesn't have uniform behavior.